store.html

```diff
@@ -1,62 +1,160 @@
-    <img src="/img/instagram-icon.svg" alt="instagram icon" width="30" height="30">
-  
-  </div>
-  </section>
-  <footer class="inc">
-    <p>@Karma Mobility,Inc</p>
-  </footer>
+    -->
+    <header>
+      <nav class="topnav">
+        <div class="content">
+          <ul class="link">
+            <li class="active"><a href="index.html">Meet Karma</a></li>
+            <li>How it Works</li>
+            <li><a href="store.html">Store</a></li>
+            <li>Blog</li>
+            <li>Help</li>
+            <li>Log in</li>
+          </ul>
+          <img
+            class="logo"
+            src="img/karma-logo.svg"
+            alt="karma-logo"
+            width="26"
+          />
+        </div>
+      </nav>
+    </header>
+    <!-- middle section -->
+
+    <main>
+      <section>
+          
+
+        <div class="mainGrid">
+
+
+            <!-- LHS contains the form on the left -->
+            <div class="LHS" >
+                <h3>Order your Karma wifi device today!</h3>
+                    <form class="row g-3">
+                        <div class="col-md-6">
+                          <label for="fname" class="form-label">First Name*</label>
+                          <input type="name" class="form-control" id="fname" required>
+                        </div>
+                        <div class="col-md-6">
+                          <label for="Lname" class="form-label">Last Name*</label>
+                          <input type="name" class="form-control" id="lname" required>
+                        </div>
+                        <div class="col-12">
+                          <label for="inputAddress" class="form-label">Address 1*</label>
+                          <input type="text" class="form-control" id="inputAddress" required>
+                        </div>
+                        <div class="col-12">
+                          <label for="inputAddress2" class="form-label">Address 2</label>
+                          <input type="text" class="form-control" id="inputAddress2">
+                        </div>
+                        <div class="col-md-8">
+                            <label for="inputcity" class="form-label">City*</label>
+                            <select id="inputcity" class="form-select" required>
+                              <option selected>Select your city...</option>
+                              <option>Sheffield</option>
+                              <option>London</option>
+                              <option>Manchester</option>
+                              <option>Leeds</option>
+                            </select>
+                          </div>
+                          <div class="col-md-4">
+                            <label for="inputZip" class="form-label">Postcode*</label>
+                            <input type="text" class="form-control" id="inputZip">
+                          </div>
+                          <div class="radios">
```

Comment:
You should style the radio and checkbox to work properly with the colour in the Figma design.

Reply:
Thank you